panel/src/components/Forms/Field/EntriesField.vue

```diff
@@ -230,7 +230,7 @@ export default {
 		value: {
 			handler(entries) {
 				// no need to add ids again if the values are the same
-				if (entries === this.values) {
+				if (entries === this.values || Array.isArray(entries) === false) {
```

Comment:
`Question:` Based on the `value` prop type this should only be an array. So I would think the only other value but an array is undefined/null. Which then we say defaults to an empty array in the prop definition. Which is then why I am wondering if instead of ignoring this new input here, we should rather set `this.entries = []`?